cryptography-core/src/cdd_claim/cdd_claim_data.rs

```diff
@@ -29,6 +33,59 @@ impl CddClaimData {
     }
 }
 
+impl Encode for CddClaimData {
+    #[inline]
+    fn size_hint(&self) -> usize {
+        32 + 32
```

Comment:
Avoid "Magic number", please use constants or `size_hint` of each element.

Reply:
Could we just define a common constant? Something like SCALAR_SIZE and RISTRETTO_POINT_SIZE. 
@ParnianAlimi

Reply:
@fmiguelgarcia That is precisely what I did.